pleskdistup/actions/systemd.py

```diff
@@ -102,7 +102,9 @@ def __init__(self):
             "sw-cp-server.service",
             "sw-engine.service",
         ]
-        self.plesk_systemd_services = [service for service in plesk_known_systemd_services if systemd.is_service_exists(service)]
+        self.plesk_systemd_services = [
+            service for service in plesk_known_systemd_services if systemd.is_service_startable(service)
```

Comment:
Apparently, already disabled services should be skipped too.

Reply:
We can skip disabled services during the prepare stage, but we shouldn't skip them during the finishing stage. If we do, they will remain disabled and stopped, which will make Plesk unavailable. Since systemd is fine with disabling already disabled services, I suggest leaving it as it is.
I can see the issue when we enable disabled services during conversion/upgrade. However, it doesn't seem like a big problem at the moment. Additionally, I don't want to make more changes in the scope of this PR. We could create an issue to address the problem